### ISSUE TITLE
The dataset is not divided

### ISSUE THREAD

**Issue by haomingHu-dd** [valeoai/RADIal#65]
Why in the 2-test.py file, the training set, test set and validation set are not divided when loading the data, but there are divisions in 1-train.py and 3-Evaluation.py
